exercises/book-store/BookStore.cs

```diff
@@ -3,7 +3,7 @@
 
 public static class BookStore
 {
-    public static double Total(IEnumerable<int> books)
+    public static decimal Total(IEnumerable<int> books)
```

Comment:
Nice! Could you also update the `Example.cs` implementation (which is in the same directory)?